src/components/KeyboardShortcuts/index.tsx

```diff
@@ -162,37 +176,34 @@ export class KeyboardShortcutsBase extends React.Component<Props> {
     _document.removeEventListener('keydown', this.keydownListener);
   }
 
-  render() {
-    const { compareInfo } = this.props;
-    const diffKeysClassName = !compareInfo ? styles.disabled : '';
+  makeClassNameForKey(key: string) {
+    // diff keys
```

Comment:
It may seem trite but I prefer to see comments written as complete sentences. It's not so easy to look at an incomplete sentence and guess what the meaning was.